build_all.sh

```diff
@@ -40,7 +40,7 @@ do
 			echo "  $(date +%T) :    Building Docker Images for Version ${ver} ${vm} ${package}  "
```

Comment:
Need to add ${runtype} in the echo statement as well

Reply:
Thanks for the review @dinogun .Added `runtype` in echo statement